packages/opencensus-instrumentation-ioredis/README.md

```diff
@@ -0,0 +1,15 @@
+# OpenCensus ioredis Instrumentation for Node.js
+[![Gitter chat][gitter-image]][gitter-url]
+
+OpenCensus Redis Instrumentation allows the user to automatically collect trace data.
+This plugin only support the `ioredis` module starting from the version 2 to latest.
+
+This project is still at an early stage of development. It's subject to change.
+
+## Useful links
```

Comment:
Nit: do you need a blank line after this heading so that the bulletet list is formatted correctly?

Reply:
Can you add ioredis link here ?